### PR TITLE
Add property model, controller and routing

### PR DESCRIPTION
**Pull request by Adi30531**

### CODE REVIEW

File: postsPhotos/post id/photo 1
Comment (yonatanbeery) [inline postsPhotos/post id/photo 1:1]: whats the point of this file

File: postsPhotos/post id/photo 2
Comment (yonatanbeery) [inline postsPhotos/post id/photo 2:1]: whats the point of this file?

File: app.rest [hunk 1]
```diff
@@ -9,15 +9,25 @@ GET http://localhost:3000/student?name=John%20Doe
 
 ###
 
-GET http://localhost:3000/student/1234567891
+GET http://localhost:3000/property/1234567891
```
Comment (yonatanbeery) [inline app.rest:12]: whats the point of this line?

File: app.rest
Comment (yonatanbeery) [inline app.rest:1]: whats the point of this file?

File: postsPhotos/post id/photo 1
Comment (Adi30531) [reply postsPhotos/post id/photo 1:1]: a mock to show how to save users photos

File: postsPhotos/post id/photo 2
Comment (Adi30531) [reply postsPhotos/post id/photo 2:1]: same

File: app.rest
Comment (Adi30531) [reply app.rest:1]: u can use it instead Postman

**Pull request merged**